app/views/mentor/dashboard/_solution.html.haml

```diff
@@ -6,7 +6,7 @@
 =link_to [:mentor, solution], class: klass do
   .core Core
   .side Side
-  .independent IM
+  .independent PM
```

Comment:
This makes me think about how we should name CSS classes moving forward as renaming concepts such as this one will make it out of sync.

Reply:
Maybe yeah. On the flipside, it's just a grep/replace job to replace them all, and it's been stable for 12months, so I'm not sure it's a big deal. I'd rather keep CSS classes as descriptive of what they do, and I think it'll be unusual for our concepts to change names.